consensus/polybft/fsm.go

```diff
@@ -504,6 +504,13 @@ func (f *fsm) getCurrentValidators(pendingBlockState *state.Transition) (Account
 		return nil, fmt.Errorf("failed to retrieve validator set for current block: %w", err)
 	}
 
+	var buf bytes.Buffer
```

Comment:
Do we need this log? I suppose you added it for testing?

Reply:
Yes, it is pretty convenient for troubleshooting purposes. 

I've added conditional logging (only if debug logs are enabled, we are going to do this stuff):
https://github.com/0xPolygon/polygon-edge/pull/1075/commits/5630d99cd301202e3a0786e081613d864869e647